.github/workflows/test_valgrind.yml

```diff
@@ -4,6 +4,10 @@ on:
     branches:
       - develop
       - master
+  pull_request:
+    branches:
+      - develop
```

Comment:
I think we removed it before for merges to develop because it was taking rather long. But also fine to keep.